static/whiteboard.js

```diff
@@ -0,0 +1,106 @@
+const canvas = document.getElementById('whiteboard');
+const ctx = canvas.getContext('2d');
+
+let cellSize = 20; // Initial size of each cell in the grid
+let zoomLevel = 1; // Initial zoom level
+const maxZoom = 3; // Maximum allowed zoom level
+const minZoom = 0.5; // Minimum allowed zoom level
+
+// Function to draw the grid with the current cell size and offset
+function drawGrid() {
+    ctx.clearRect(0, 0, canvas.width, canvas.height); // Clear the canvas
+    const scaledCellSize = cellSize * zoomLevel; // Adjusted cell size based on zoom level
+
```

Comment:
Use a single loop to draw both vertical and horizontal lines

Reply:
For now I don't know how to do it in a single loop but I am grateful for your consideration